engine/CMakeLists.txt

```diff
@@ -15,6 +16,7 @@ target_link_libraries(engine
     PUBLIC
         render
         gameobjects
+        fmt
```

Comment:
I would guess fmtlib has a `fmt::fmt` style.

When using the "fully qualified" name of a library, it helps you debug if something goes wrong, so they're generally recommended.